pkg/provisioner/provisioner.go

```diff
@@ -85,7 +85,7 @@ type Provisioner interface {
 
 	// PowerOff ensures the server is powered off independently of any image
 	// provisioning operation.
```

Comment:
We should add something in the comment about the boolean, something like `the boolean argument may be used to specify if a hard-reboot (force power-off) is required`

Reply:
Agreed. I've taken care of this.